wp-content/themes/core/assets/css/src/theme/actions/_mixins.pcss

```diff
@@ -262,3 +262,40 @@
 		}
 	}
 }
+
+/* -----------------------------------------------------------------------------
+ * Button: Close
+ * ----------------------------------------------------------------------------- */
+
+@define-mixin a-close {
```

Comment:
Some white space issues here. A `gulp lint` in the theme directory before merging should ix this and any other small issues throughout.